test/html.test.js

```diff
@@ -930,3 +930,133 @@ function() {
     'active source buffers ends with audio source buffer');
 
 });
+
+QUnit.test('calls changeInfo_ on mediaSource when info is given for audio', function() {
```

Comment:
Seems like this is covered by the test below. Any background on why the extra check is needed?